bioio_base/io.py

```diff
@@ -62,3 +64,38 @@ def pathlike_to_fs(
     # any open file buffers _after_ any API call. API calls must themselves call
     # fs.open and complete their function during the context of the opened buffer.
     return fs, path
+
+
+def search_for_node(
```

Comment:
is this used anywhere? Could delete?

Reply:
Its used for lif and czi files

Reply:
Though we may be able to limit it to just lif

Reply:
removed in [d739331](https://github.com/bioio-devs/bioio-base/pull/29/commits/d739331e5756e035e795113a19359223f54f5b81) and moved to lif